tests/zfs-tests/include/libtest.shlib

```diff
@@ -2629,6 +2629,38 @@ function get_config
 	fi | awk -F: -v cfg="$config:" '$0 ~ cfg {sub(/^'\''/, $2); sub(/'\''$/, $2); print $2}'
 }
 
+#
+# Get random number between min and max number.
+#
+# $1 Minimal value
+# $2 Maximal value
+# [$3 How many]
+#
+function random
+{
+	typeset -ri min=$1
+	typeset -ri max=$2
+	typeset -i value
+	typeset -i count
+	typeset -i i
+
+	if [[ -z "$3" ]]; then
+		count=1
+	else
+		count=$3
+	fi
+
+	for (( i = 0; i < $count; i++ )); do
+		while true; do
+			((value = RANDOM % (max + 1)))
+			if ((value >= min)); then
+				break
+			fi
+		done
+		echo $value
+	done
+}
```

Comment:
There an alternate version of this function already in `zfs-tests/include/math.shlib` called `random_int_between`.  Let's unify them and extend the existing version to take that third "How Many" option.

```sh

# A simple function to get a random number between two bounds (inclusive)
#
# Note since we're using $RANDOM, $min+32767 is the largest number we
# can accept as the upper bound.
#
# $1 lower bound
# $2 upper bound
function random_int_between
{
        typeset -i min=$1
        typeset -i max=$2

        echo $(( (RANDOM % (max - min + 1)) + min ))
}
```